Tests/Functional/Drivers/Queue/AbstractQueueDriverTest.php

```diff
@@ -54,147 +50,18 @@ protected function tearDown()
     public function testSendShouldNotChangeMessage(MessageInterface $msg)
     {
         $clone = clone $msg;
-
+        if (!$this->driver->isConnected()) {
+            $this->driver->connect();
+        }
         $this->driver->send($this->createQueueMessage($msg));
 
         $this->assertSame(serialize($clone), serialize($msg));
         $this->driver->send($this->createQueueMessage($msg));
     }
 
     /**
-     * @dataProvider getMessages
-     *
-     * @param MessageInterface $msg
-     */
-    public function testShouldSendReceiveAndAckOnce(MessageInterface $msg)
```

Comment:
Is this test not needed anymore? I saw the rest of test cases in this class where moved to Tests/Functional/Drivers/Queue/StompQueueDriverTest.php but this one no

Reply:
The tests that were moved to the Stomp driver are just the ones that has some specific function needed as subscribe. This test is used by the both drivers.

Reply:
If this test is used for both drivers shouldn't be maintained here?

Reply:
Yes, because this is the abstract class for both drivers.

Reply:
:-1: sooooo, bring it back?

Reply:
Yes, we should it bring back

Reply:
I have brought this test back in StompQueueDriverTest